esp-link/cgiwifi.c

```diff
@@ -26,6 +27,27 @@ Cgi/template routines for the /wifi url.
 #define DBG(format, ...) do { } while(0)
 #endif
 
+#define LOG(severity, format, ...)  do {	\
+	char buffer[128];			\
+	os_sprintf(buffer, format, ## __VA_ARGS__);	\
```

Comment:
Something nice that the os_printf macro does is store the format strings in flash to save on RAM. I don't think os_sprintf does this...